tests/product/serializers/test_discount_validate.py

```
@@ -0,0 +1,21 @@
+import pytest
+from product.serializers import ProductWriteSerializer
+
+
+def test_invalid_discount_str():
+    serializer = ProductWriteSerializer()
+
+    with pytest.raises(Exception):
```

Ruff (0.8.2)

`pytest.raises(Exception)` should be considered evil

(B017)